### PR TITLE
polymc: Use portable version instead

### PR DESCRIPTION
**Pull request by Pato05**
## What package release type is this?

> - [x] stable
> - [ ] beta
> - [ ] dev/nightly/canary

### CODE REVIEW

File: bucket/polymc.json [hunk 1]
```diff
@@ -3,19 +3,36 @@
...
+    ],
     "checkver": {
         "github": "https://github.com/PolyMC/PolyMC"
     },
     "autoupdate": {
-        "url": "https://github.com/PolyMC/PolyMC/releases/download/$version/PolyMC-Windows-$version.zip"
+        "url": "https://github.com/PolyMC/PolyMC/releases/download/$version/PolyMC-Windows-Portable-$version.zip"
     }
-}
+}
```
Comment (Calinou) [inline bucket/polymc.json:38]:
```suggestion
}

```

**Review by Calinou — APPROVED** [review]
Thanks! Congratulations for your first merged pull request :tada:

**Comment by dada513** [comment]
why?

**Comment by DioEgizio** [comment]
just why

**Comment by Calinou** [comment]
> just why

Scoop aims to use portable apps in manifests whenever possible, as they don't modify files outside the location they're explicitly written in. The issue is that switching from non-portable to portable breaks existing setups, and I'm not sure if it's worth the trouble for scoop-games (where most manifests are non-portable anyway).

**Comment by Pato05** [comment]
> > just why
> 
> Scoop aims to use portable apps in manifests whenever possible, as they don't modify files outside the location they're explicitly written in.
> 
> The issue is that switching a manifest from non-portable to portable breaks existing setups, and I'm not sure if it's worth the trouble for scoop-games (where most manifests are non-portable anyway).

Yeah, I actually didn't think of that before, wouldn't it be possible to migrate from the non-portable build with a pre-install script, maybe?

**Pull request merged**